app/robots/iCubGenova04/oculusConfig.ini

```diff
@@ -22,6 +22,12 @@ playerOrientationThreshold    0.2
 humanHeight                   1.76
 # The robot arm span is the distance between the left and right index fingertips when the arms are aligned, i.e. T pose.
 robotArmSpan                  1.10
+# If resetCameras !=0, the module will attempt to reset the cameras during the initialization phase.
```

Comment:
Well, in principle that's not true. As a matter of fact, the value is a char, since it is in a text file. It is then interpreted as a boolean. This means that every value different from zero will be considered ``true``. So, 
> ``If resetCameras =1, the module will attempt to reset the cameras during the initialization phase.``

would not be precise, as also ``resetCameras =7`` would have the same effect.

Reply:
OK, make sense. So, in `    bool resetCameras = generalOptions.check("resetCameras", yarp::os::Value(false)).asBool();` if the char value is not `0`, the behavior of the function instantiate `resetCameras` with true value, right? @S-Dafarra

Reply:
Yes, as per the [documentation](http://www.yarp.it/git-master/classyarp_1_1os_1_1Value.html#a3f49bd86a14537ca63f59297513e80eb)
> boolean value if value is indeed a boolean. If it is another type, the appropriate cast value is returned. Otherwise returns false.

and basically, everything that is different from 0 or ``false`` is casted to ``true``.